### PR TITLE
Update to 0.19

### PR DESCRIPTION
**Pull request by kraklin**
Only did elm-upgrade and edit the files to be able to build and run tests (with elm-verify-example as well). All were green ;)

I bumped the version from 1.2.1 to 1.2.2 but feel free to change it as you wish.

### CODE REVIEW

File: src/Set/Extra.elm [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-        , subset
-        , toggle
-        )
+module Set.Extra exposing (concatMap, filterMap, subset, toggle)
 
 {-| Convenience functions for working with Set.
 
 @docs concatMap, filterMap, subset, toggle
 
-    -- This module is used in the examples
```
Comment (kraklin) [inline src/Set/Extra.elm:13]: I was unable to add `import Set exposing (Set)` globally. It won't generate right tests from that :/ 

Any trick here?

File: tests/Tests.elm [hunk 2]
```diff
@@ -93,14 +94,14 @@ all =
             [ test "Applies a function that may succeed to all values in the list, but only keep the successes." <|
                 \() ->
                     Set.fromList [ "1", "2", "3", "hello", "4", "world" ]
-                        |> Set.Extra.filterMap (String.toFloat >> Result.toMaybe)
```
Comment (kraklin) [inline tests/Tests.elm:96]: Also `String.toFloat` now returns Maybe instead of `Result` ;)

File: src/Set/Extra.elm (hunk 1 above)
Comment (stoeffel) [reply src/Set/Extra.elm:13]: Nope, we remove support for global imports. Because it's nice to see all relevant imports next to the examples.

**Comment by stoeffel** [comment]
looks great! thanks ❤️

**Comment by stoeffel** [comment]
published with 1.2.3

**Pull request merged**